docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,191 @@
+- Examples for the event listeners.
+- Documentation and examples for selectors.
+- Add the ability to export and restore some parts of the grid state.
+
+#### Synchronous state initialization
+
+In previous versions, the state was first populated with default values, and then in a `useEffect`, given the values provided as props (`props.pageSize` for instance), or derived from the props (the sorted and filtered rows derived from the `props.rows`, `props.sortModel` and `props.filterModel`).
+This was causing an additional render with useless data, and the we had to be careful to avoid flickering between the fake and real data.
+In v5, the state is initialized synchronously during the first render.
+
+Note that for now, the state updates coming from controlled props are still asynchronous.
+If you pass `props.pageSize`, we will apply it to the state in a `useEffect`, and therefore if you read the state just after the render (for instance in a `useLayoutEffect`), you will still see the old version.
+
+## Reduced style specificity for easier customization
+
+In previous versions most of the various `DataGrid` and `DataGridPro` components had a CSS specificity of 2, meaning that style overrides and customizations were harder, requiring you to look at the DOM tree in order to pick the correct selector.
+With MUI X v5 we have reduced the CSS specificity of most of the internal `DataGrid` and `DataGridPro` components to 1.
+This will enable developers to more easily change the look and feel of the grid's components.
+
+**Before**
+
+```jsx
+const useStyles = makeStyles(() => ({
+  root: {
+    '& .MuiDataGrid-toolbarContainer': {
+      padding: 50,
+    },
+  },
+}));
+
+const MyCustomToolbar = () => {
+  return <GridToolbarContainer>My custom toolbar</GridToolbarContainer>;
+};
+
+export default function App() {
+  const classes = useStyles();
+
+  return (
+    <div style={{ height: 400, width: '100%' }} className={classes.root}>
+      <DataGrid
+        components={{
+          Toolbar: MyCustomToolbar,
+        }}
+      />
+    </div>
+  );
+}
+```
+
+**After**
+
+```jsx
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 50,
+  },
+}));
+
+const MyCustomToolbar = () => {
+  const classes = useStyles();
+
+  return (
+    <GridToolbarContainer className={classes.root}>
+      My custom toolbar
+    </GridToolbarContainer>
+  );
+};
+
+export default function App() {
+  return (
+    <div style={{ height: 400, width: '100%' }}>
+      <DataGrid
+        components={{
+          Toolbar: MyCustomToolbar,
+        }}
+      />
+    </div>
+  );
+}
+```
+
+#### Limitations
+
+Although this was a clear improvement, we still had to keep a style specificity of 2 for some parts of the `DataGrid` and `DataGridPro`, specifically the `GridColumnHeaderItem`, `GridRow` and `GridCell` along with all of the components that are nested in them.
+This is due to performance implications related to how Emotion injects styles into the page,
```

Comment:
The official capitalization, at least, from what I understood the last time I asked them:

```suggestion
This is due to performance implications related to how [emotion](https://emotion.sh/) injects styles into the page,
```